authorization/v2/parser.go

```diff
@@ -0,0 +1,86 @@
+package v2
```

Comment:
I think it might be better to store this logic in [uniformResourceIdentifier.go](https://github.com/unicsmcr/hs_auth/blob/master/authorization/v2/uniformResourceIdentifier.go)  as this file contains logic for the `UniformResourceIdentifier` struct

Reply:
Yep, would fit better there :)